test/Microsoft.Azure.Devices.Edge.Test.Common/config/EdgeConfigBuilder.cs

```diff
@@ -167,6 +171,25 @@ ModuleConfiguration BuildEdgeAgent(IEnumerable<ModuleConfiguration> configs)
             return agentBuilder.Build();
         }
 
+        bool IsSystemModule(ModuleConfiguration module)
+        {
+            return module.Name.Equals("$edgeHub") || module.Name.Equals("$edgeAgent");
```

Comment:
Check with Damon if he have any concerns to reference EdgeAgent.Core project for these constants.